l3_footstep_planning_msgs/PatternGeneratorParameters params
---
l3_footstep_planning_msgs/ErrorStatus status
